test/puppetlabs/puppetdb/cli/services_test.clj

```diff
@@ -65,13 +66,19 @@
   (svc-utils/with-single-quiet-pdb-instance
     (let [dispatcher (get-service svc-utils/*server* :PuppetDBCommandDispatcher)
           query-fn (partial query (get-service svc-utils/*server* :PuppetDBServer))]
-      (enqueue-command dispatcher :replace-facts 4
-                       {:certname "foo.local"
-                        :environment "DEV"
-                        :values {:foo "the foo"
-                                 :bar "the bar"
-                                 :baz "the baz"}
-                        :producer_timestamp (to-string (now))})
+      (enqueue-command dispatcher
```

Comment:
This code is stale, I have some test utility code that accepts a clojure data structure and coerces to a stream, switching to that would simplify this code and the test below it